source/fundamentals/connection.txt

```diff
@@ -0,0 +1,235 @@
+================
+Connection Guide
+================
+
+This guide shows you how to connect to a MongoDB instance or replica set
+deployment using the Node.js driver.
+
+--------------
+Connection URI
+--------------
+
+The **connection URI** is the set of instructions that the driver uses to
+connect to a MongoDB instance or replica set deployment. It instructs the
+driver on how it should connect to MongoDB and how it should behave while
+connected.
+
+The following snippet is an example connection URI string.
+
+.. code-block::
+
+     mongodb+srv://sample-hostname:27017/?poolSize=20&useUnifiedTopology=true
+
+The first part identifies the connection string format. In the example above,
+we use ``mongodb+srv`` which specifies the :manual:`DNS Seedlist Connection
+Format </reference/connection-string/#dns-seedlist-connection-format>` which
+indicates that the hostname following it corresponds to the DNS SRV record of
+your MongoDB instance or deployment. If your instance of deployment does not
+have a DNS SRV record, use ``mongodb`` to specify the
```

Comment:
Can we capitalize 'mongodb' to 'MongoDB' for consistency?
this should read:
``If your instance of deployment does not
have a DNS SRV record, use ``MongoDB`` to specify the ... ``

Reply:
Are you sure that would be consistent? If anyone has been using `MongoDB://` in the connection URI, we should probably change it to use the same lowercase syntax as in the specification:
https://github.com/mongodb/specifications/blob/master/source/initial-dns-seedlist-discovery/initial-dns-seedlist-discovery.rst#seedlist-discovery

Reply:
Sorry I misinterpreted this sentence - you're right, it would not be `MongoDB://` it should be `mongodb://` as it currently is.